mage_ai/server/kernel_output_parser.py

```diff
@@ -44,6 +46,9 @@ def parse_output_message(message: dict) -> dict:
         text_stdout = content.get('text')
         data_content = text_stdout.split('\n')
         data_type = DataType.TEXT_PLAIN
+        if content.get('name') == 'stdout' and len(data_content) > PRINT_OUTPUT_MAX_LINES:
```

Comment:
The websocket server could send multiple messages to UI continuously. Limiting the number of rows in one message doesn't solve the issue completely.

Reply:
@wangxiaoyou1993 Added a [commit](https://github.com/mage-ai/mage-ai/pull/2659/commits/ab1511d0fd833edaf46b5fcee589a24ff4c79d04) that limits the output that would be saved to the pipeline (that should solve the issue of an excessively large req body due to a large print output) even if multiple messages with print output is sent from BE.

For example in the attached gif, I temporarily changed the `MAX_PRINT_OUTPUT_LINES` constant to `100` for the demo test. The print output sent from the BE is 101 lines, so the print output is not included in the req body when saving the pipeline (but the sample data for the table is still saved and visible after refresh, while the print output is not).
![limited print output saved](https://github.com/mage-ai/mage-ai/assets/78053898/9bfee449-7112-470b-a46f-4883332d453a)